app/wizards/claims/add_mentor_wizard.rb

```diff
@@ -0,0 +1,38 @@
+module Claims
+  class AddMentorWizard < BaseWizard
+    attr_reader :school
+
+    def initialize(school:, params:, state:, current_step: nil)
+      @school = school
+      super(state:, params:, current_step:)
+    end
+
+    def define_steps
+      add_step(MentorStep)
+      if steps[:mentor].mentor.nil?
```

Comment:
@Kizr pointed this out to me other day.
```suggestion
      if steps.fetch(:mentor).mentor.nil?
```